src/rust/rust_kvs/tests/cit_default_values.rs

```diff
@@ -0,0 +1,412 @@
+//! Persistency tests.
+//!
+//! Requirements verified:
+//! - Default Values (feat_req__persistency__default_values)
+//! The KVS system shall support predefined default values for keys.
+//! - Default Values Retrieval (feat_req__persistency__default_value_get)
+//! The KVS system shall support retrieving the default value associated with a key.
+//! - Set default key values via file (feat_req__persistency__default_value_file)
+//! The KVS shall support the configuration of default key values using an external file.
+//! - Intra-Process Data Access (feat_req__persistency__intra_process_comm)
```

Comment:
I meant - this is basically same as multiple KVS req - move it to `cit_multiple_kvs.rs`.